lynis: update to 3.0.7.

jhe2 opened the pull request:
<!-- Uncomment relevant sections and delete options which are not applicable -->

#### Testing the changes
- I tested the changes in this PR: **YES**

<!--
#### New package
- This new package conforms to the [quality requirements](https://github.com/void-linux/void-packages/blob/master/Manual.md#quality-requirements): **YES**|**NO**
-->

<!-- Note: If the build is likely to take more than 2 hours, please [skip CI](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#continuous-integration)
and test at least one native build and, if supported, at least one cross build.
Ignore this section if this PR is not skipping CI.
-->
<!-- 
#### Local build testing
- I built this PR locally for my native architecture, (ARCH-LIBC)
- I built this PR locally for these architectures (if supported. mark crossbuilds):
  - aarch64-musl
  - armv7l
  - armv6l-musl
-->

paper42 reviewed (commented):
* add `$changelog`
* all the vinstalls to usr/share/doc/ seem useless to me (maybe except for FAQ), what would you think about removing them?

jhe2 commented:
> * add `$changelog`
> 
>     * all the vinstalls to usr/share/doc/ seem useless to me (maybe except for FAQ), what would you think about removing them?

I have added the changelog URL and removed the ``INSTALL``, ``README``  and ``CHANGELOG.md`` files from ``/usr/share/doc/lynis``, but kept ``FAQ``.

I've noticed that other packages sometimes also contain CHANGELOG files in their ``/usr/share/doc`` directories. Is it generally discouraged to include them there?

paper42 commented on srcpkgs/lynis/template:
```diff
@@ -19,9 +20,6 @@ do_install() {
 	vbin lynis
 	vman lynis.8
 	vmkdir usr/share/doc/lynis
```
```suggestion
```

paper42 commented on srcpkgs/lynis/template:
```diff
@@ -19,9 +20,6 @@ do_install() {
 	vman lynis.8
 	vmkdir usr/share/doc/lynis
-	vinstall CHANGELOG.md 0644 usr/share/doc/lynis
 	vinstall FAQ 0644 usr/share/doc/lynis
```
```suggestion
	vdoc FAQ
```

paper42 commented:
> > * add `$changelog`
> >   
> >   * all the vinstalls to usr/share/doc/ seem useless to me (maybe except for FAQ), what would you think about removing them?
> 
> I have added the changelog URL and removed the `INSTALL`, `README` and `CHANGELOG.md` files from `/usr/share/doc/lynis`, but kept `FAQ`.
> 
> I've noticed that other packages sometimes also contain CHANGELOG files in their `/usr/share/doc` directories. Is it generally discouraged to include them there?

From what I can see, CHANGELOG is explicitly included only in 4 packages (durden fingerprint-gui go-ipfs hunspell-it_IT). I don't think it's necessary to include a changelog, it doesn't add any value to the documentation and people can check it online if they really want to.

jhe2 commented:
> From what I can see, CHANGELOG is explicitly included only in 4 packages (durden fingerprint-gui go-ipfs hunspell-it_IT). I don't think it's necessary to include a changelog, it doesn't add any value to the documentation and people can check it online if they really want to.

Of the packages installed on my system, the ``inxi`` package also comes with a changelog, so does ``flac`` (although in case of flac it appears to be installed implicitly).
I actually like not having to look online for things like that, but I'm all for consistency, so if such files are not supposed to be included, that's fine. I guess it is not of much use, if only some packages include a CHANGELOG files and other don't.
